### DISCUSSION TITLE
Toggle List

### DISCUSSION THREAD

**Discussion by stupididi** [outline/outline#1398] [General]
Could it be possible to add a Toggle List feature similar to those in Notion, please? Thank you!

Attached is an example.
![Toggle List Example](https://user-images.githubusercontent.com/68925613/88722543-0badfd80-d0ed-11ea-9a72-175ed11729b2.gif)

**Comment by tommoor** [comment]
This is fun, I've not seen this one before. We need to restore toggleable headings which will go someway towards helping with this usecase too.  I'll take this opportunity to note that the editor is totally OSS:

https://github.com/outline/rich-markdown-editor

**Reply by robin-schaerer** [reply]
Need this so much!!!

**Comment by stupididi** [comment]
Okay, thanks so much!

**Comment by adapowers** [comment]
+1 here—Outline is great so far, but I was really hoping for this feature since a lot of my Notion docs make heavy use of it. It’s a really great way to include optional information and hide details.
